src/applications/vaos/utils/timezone.js

```diff
@@ -14,6 +14,19 @@ const TIMEZONE_LABELS = {
   AT: 'Atlantic time',
 };
 
+const GMT_TABLE_MAPPING = {
+  'GMT+8': 'PT', // Asia/Manila
```

Comment:
FYI, it was mentioned that this should be PHT during the last meeting.

Also it seems like none of these timezones use daylight savings (e.g. https://www.timeanddate.com/time/change/philippines) so I think these mappings should be sufficient.